Description: remove pkey_mutex 
 From linux 01690e9:
 .
 The last user of pkey_mutex was removed in db84f8803759 ("IB/ipoib: Use
 P_Key change event instead of P_Key polling mechanism") but the lock
 remained.
 This patch removes it.
Author: Brian T. Smith <bsmith@systemfabricworks.com>
Author: Sebastian Andrzej Siewior <bigeasy@linutronix.de>
Origin: https://github.com/torvalds/linux/commit/01690e9c70c0b42072d6c82470d78b747fcc5c70#diff-354653dff7ec9ddd6ac73f85f2956316
Forwarded: not-needed
Last-Update: <2017-12-14>
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/ib_ipoib/ipoib_ib.c
+++ b/ib_ipoib/ipoib_ib.c
@@ -51,8 +51,6 @@
 		 "Enable data path debug tracing if > 0");
 #endif
 
-static DEFINE_MUTEX(pkey_mutex);
-
 struct ipoib_ah *ipoib_create_ah(struct net_device *dev,
 				 struct ib_pd *pd, struct ib_ah_attr *attr)
 {
